EasyPost.Tests/InsuranceTest.cs

```diff
@@ -0,0 +1,145 @@
+using System.Collections.Generic;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace EasyPost.Tests
+{
+    [TestClass]
+    public class InsuranceTest
+    {
+        private Dictionary<string, object> _fromAddressData { get; set; }
+        private Insurance _insurance { get; set; }
+        private Dictionary<string, object> _parcelData { get; set; }
+        private Dictionary<string, object> _toAddressData { get; set; }
```

Comment:
Does the linter want these ordered alphabetically? If not, it'd make more sense to have from/to address next to each other, though I assume they're this way for the linter and that's fine.